src/meta/protos/proto/catalog.proto

```diff
@@ -52,7 +52,7 @@ message HiveCatalogOption {
   uint64 min_reader_ver = 101;
 
   // Hive metastore thrift uri
-  string address = 1;
+  string hive_metastore = 1;
```

Comment:
NOTE: this is a breaking change. We should add a new field in proto instead of change it directly. I prefer to keep using `address` internally and only change the sql represents.

Reply:
Not sure change the name will break in proto, cc @drmingdrmer

Reply:
It's compatible unless the field index(`1`) is changed